build/src/push.js

```diff
@@ -84,7 +94,26 @@ async function pushImage(definitionId, repo, release, updateLatest,
             // Determine tags to use
             const versionTags = configUtils.getTagList(definitionId, release, updateLatest, registry, registryPath, variant);
             console.log(`(*) Tags:${versionTags.reduce((prev, current) => prev += `\n     ${current}`, '')}`);
-
+            let architectures = configUtils.getBuildSettings(definitionId).architectures;
+            switch (typeof architectures) {
+                case 'string': architectures = [architectures]; break;
+                case 'object': if (!Array.isArray(architectures)) { architectures = architectures[variant]; } break;
+                case 'undefined': architectures = ['linux/amd64']; break;
+            }
+            console.log(`(*) Target image architectures: ${architectures.reduce((prev, current) => prev += `\n     ${current}`, '')}`);
+            let localArchitecture = process.arch;
+            switch(localArchitecture) {
+                case 'arm': localArchitecture = 'linux/arm/v7'; break;
+                case 'aarch32': localArchitecture = 'linux/arm/v7'; break;
+                case 'aarch64': localArchitecture = 'linux/arm64'; break;
+                case 'x64': localArchitecture = 'linux/amd64'; break;
+                case 'x32': localArchitecture = 'linux/386'; break;
```

Comment:
is this supposed to be `linux/i386`?  Doesn't look right to me without the `i` for some reason, but I might just be wrong

Reply:
Yeah for Docker it's w/o the `i`.  Type `docker buildx ls` locally to see architectures that are setup to emulate. Desktop has them all installed.